src/course-unit/sidebar/components/sidebar-footer/ActionButtons.jsx

```diff
@@ -0,0 +1,51 @@
+import { useSelector } from 'react-redux';
+import { Button } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { getCourseUnitData } from '../../../data/selectors';
+import messages from '../../messages';
+
+const ActionButtons = () => {
+  const intl = useIntl();
+  const {
+    published,
+    hasChanges,
+    enableCopyPasteUnits,
+  } = useSelector(getCourseUnitData);
+
+  return (
+    <>
+      {/* TODO: Sidebar functionality will be added to: https://youtrack.raccoongang.com/issue/AXIMST-24 */}
+      {(!published || hasChanges) && (
+        <Button
+          className="mt-3.5"
+          variant="outline-primary"
+          size="sm"
+        >
+          {intl.formatMessage(messages.actionButtonPublishTitle)}
+        </Button>
+      )}
+      {(published && hasChanges) && (
+        <Button
+          className="mt-2"
+          variant="link"
+          size="sm"
+        >
+          {intl.formatMessage(messages.actionButtonDiscardChangesTitle)}
+        </Button>
+      )}
+      {/* TODO: Unit copying functionality will be added to: https://youtrack.raccoongang.com/issue/AXIMST-375 */}
```

Comment:
We will open this PR into upstream. I think it should not contain such comments that are related to our work to make sure PRs are merged faster without additional questions.

Reply:
Removed